### PR TITLE
Updating Actions reference for MFTF 2.3.13

### PR DESCRIPTION
**Pull request by dobooth**
## This PR is a:
-  Content update

## Summary

When this pull request is merged, it will...

Add new PWA Actions to reference.

whatsnew
Adds two new PWA actions to the Actions reference guide for the MFTF 2.3.13 release.
[waitforPwaElementNotVisibleType](https://devdocs.magento.com/mftf/2.3/test/actions.html#waitforpwaelementnotvisibletype)
[waitforPwaElementVisibleType](https://devdocs.magento.com/mftf/2.3/test/actions.html#waitforpwaelementvisibletype)

### CODE REVIEW

File: mftf/2.3/test/actions.md [hunk 1]
```diff
@@ -2529,6 +2529,83 @@ Attribute|Type|Use|Description
 <!-- Wait up to 30 seconds for the current page to fully load before continuing. -->
 <waitForPageLoad stepKey="waitForPageLoad"/>
 ```
+### waitForPwaElementNotVisibleType
+
+Waits up to given time for a PWA Element to disappear from the screen.
```
Comment (jcalcaben) [inline mftf/2.3/test/actions.md]:
```suggestion
Waits up to the given `time` for a PWA Element to disappear from the screen.
```

File: mftf/2.3/test/actions.md [hunk 2]
```diff
@@ -2529,6 +2529,83 @@ Attribute|Type|Use|Description
...
+
+#### Example
+
+```xml
+<!-- Wait for the PWA element to disappear. -->
+<waitForPwaElementNotVisibleType time="1" stepKey="waitForPwaElementNotVisibleType"/>
+```
+### waitForPwaElementVisibleType
+
+Waits up to given time for a PWA Element to appear on the screen.
```
Comment (jcalcaben) [inline mftf/2.3/test/actions.md]:
```suggestion
Waits up to the given `time` for a PWA Element to appear on the screen.
```

File: mftf/2.3/test/actions.md [hunk 3]
```diff
@@ -2529,6 +2529,83 @@ Attribute|Type|Use|Description
...
+
+#### Example
+
+```xml
+<!-- Wait up to 5 seconds for the element to disappear. -->
+<waitForPwaElementNotVisibleType time="5" stepKey="waitForPwaElementNotVisibleType"/>
+```
+### waitForPwaElementVisibleType
+
+Waits for a PWA Element to appear on the screen..
```
Comment (jcalcaben) [inline mftf/2.3/test/actions.md]:
```suggestion
Waits for a PWA Element to appear on the screen.
```
Remove extra period at the end

File: mftf/2.3/test/actions.md (hunk 2 above)
Comment (dobooth) [reply mftf/2.3/test/actions.md]: Thanks!

**Comment by dobooth** [comment]
running tests

**Comment by dshevtsov** [comment]
@dobooth , in whatsnew:
- the links to sections are incorrect
- use a colon instead of a period for lists

**Pull request merged**